.gitignore

```diff
@@ -2,3 +2,4 @@ _site
 .sass-cache
 .jekyll-metadata
 assets/css/**/*.css
+Gemfile.lock
```

Comment:
@cynthiapereira did you have some issues with Gemfile.lock? He is necessary to lock versions of gems that we use on project.

Reply:
Yes, why is it being removed?

Reply:
Hey guys, I had issues in the past with Gemfile.lock + docker + jekyll that was resolved by removing it. I do not totally understand how the Gemfile works, so fell free to revert.